controls/V-92985.rb

```diff
@@ -1,6 +1,4 @@
-# encoding: UTF-8
```

Comment:
Is there a reason for removing the encoding? @aagiardd

Reply:
This was likely a cookstyle correction

Reply:
I see, thanks for adding it back in!